tests/acceptance/_bootstrap.php

```diff
@@ -1 +1,3 @@
 <?php
+define("TIMEOUT", 60);
```

Comment:
Sure. I will add the same in README.md and I will also add condition like suggested by Yves.